src/state.js

```diff
@@ -1,3 +1,12 @@
-const INITIAL_STATE = {};
+const INITIAL_STATE = {
+	locations: [],
+	currentLocation: [],
+	searchQuery: '',
+	filters:[],
```

Comment:
Also I think we agreed we would be doing filters as a dictionary so we would need to add

`
{
  isOpen: false,
  communityGardens: false,
  farmersMarkers: false,
  supermarkets: false,
}
`